mail/pop3.js

```diff
@@ -1,5 +1,5 @@
 const net = require("net");
-const mysql = require("mysql");
+const mysql = require("mysql2");
```

Comment:
Ran into some errors about unsupported authentication when using `mysql`. I dropped in `mysql2` which was already used in `smtp.js` and it worked great.